aries-site/src/examples/components/layer/LayerStickyFormExample.js

```diff
@@ -0,0 +1,164 @@
+import React, { useContext, useState } from 'react';
+import {
+  Button,
+  Box,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Layer,
+  ResponsiveContext,
+  Text,
+  TextInput,
+  TextArea,
+  Select,
+} from 'grommet';
+
+const options = ['Any', 'None', 'Any Scheduled'];
+
+export const LayerStickyFormExample = () => {
+  const [formValues, setFormValues] = React.useState({
+    monitorName: 'C2 Monitor',
+    desc: `Sustainable neutra echo park helvetica freegan
+    forage hashtag meditation kogi activated charcoal
+    pitchfork`,
+    monitorType: 'Rule',
+    frameworks: 'Any Scheduled',
+    severity: 'Any',
+    provider: 'Any',
+    account: 'Any',
+    ruleName: 'Any Scheduled',
+    email: 'jane@hpe.com',
+  });
+  const [open, setOpen] = useState(false);
+  const size = useContext(ResponsiveContext);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+
+  const onFormChange = value => {
+    setFormValues(value);
+  };
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+  return (
+    <>
+      <Box align="start">
+        <Button label="Show me the Layer" onClick={onOpen} primary />
+      </Box>
+      {open && (
+        <Layer
+          position="right"
+          full={!['xsmall', 'small'].includes(size) ? 'vertical' : true}
+          modal
+          onClickOutside={onClose}
+          onEsc={onClose}
+        >
+          <Box
+            gap="small"
+            fill="vertical"
+            overflow="auto"
+            width={!['xsmall', 'small'].includes(size) ? 'medium' : undefined}
+            pad="medium"
```

Comment:
I think we need the padding to go on the header itself and on the Form (or Box surround the form if that doesn't work). Otherwise the content is visible on scroll above the header:

<img width="385" alt="Screen Shot 2022-04-19 at 3 03 13 PM" src="https://user-images.githubusercontent.com/12522275/164109322-e931fbc5-3f50-4454-8b19-0476e1b469b3.png">

Reply:
I'm still seeing this issue @britt6612